projects/plugins/protect/src/class-rewind.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Class to handle Credentials
+ *
+ * @package automattic/jetpack-protect-plugin
+ */
+
+namespace Automattic\Jetpack\Protect;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_Options;
+use WP_Error;
+
+/**
+ * Class that handles the rewind api call.
+ */
+class Rewind {
+	/**
+	 * Get the rewind state, if no creds are set the state will be 'awaiting_for_credentials'
+	 *
+	 * @return bool
+	 */
+	public static function get_rewind_state() {
+		$blog_id      = Jetpack_Options::get_option( 'id' );
+		$is_connected = ( new Connection_Manager() )->is_connected();
+
+		if ( ! $blog_id || ! $is_connected ) {
+			return new WP_Error( 'site_not_connected' );
```

Comment:
```suggestion
			return false;
```
Based on current implementation and the `@return` doc block, this method should only return a boolean `false` on error. Though, using `WP_Error` would be the more core-like approach and probably be a good pattern to pursue.

Reply:
Yeah, WP_Error would be nice, but let's change it to false now to make it consistent to the other methods and then we can make an effort to change everything to WP_Error to be more core like in the future if we want. The key here is consistency in my opinion.